setup.py

```diff
@@ -1,6 +1,11 @@
 from distutils.core import setup
 import cffi_build.cffi_build as cffi_build
-from os.path import join
+from shutil import copyfile
```

Comment:
This stuff is done in the cffi_built script now. 
Just checkout this file to master, and it will be good to merge.